### PR TITLE
chore: ShreyaPrasad1209.json addition

### PR DESCRIPTION
**Pull request by ShreyaPrasad1209**
## Changes proposed

Added a new profile by adding my data inside ShreyaPrasad1209.json

## Check List (Check all the applicable boxes) 

- [x] My code follows the code style of this project.
- [ ] My change requires changes to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] All new and existing tests passed.
- [ ] This PR does not contain plagiarized content.
- [x] The title of my pull request is a short description of the requested changes.

## Note to reviewers

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/1469"></a>

### CODE REVIEW

**Review by emmalearnscode — APPROVED** [review]
Looks great!

**Review by kumarsonsoff3 — APPROVED** [review]
Looks good to me! 👍 
Thanks for contributing!

**Review by Panquesito7 — APPROVED** [review]
Thanks for contributing! 🎉


__
[sema-logo](https://semasoftware.com/gh) &nbsp;**Summary:** :ok_hand: This code looks good&nbsp; | &nbsp;**Tags:** Efficient

File: public/data/ShreyaPrasad1209.json [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+      "icon": "linkedin"
+    },
+    {
+      "name": "Follow me on Medium",
+      "url": "https://medium.com/@shreyapd06",
+      "icon": "medium"
+    },
+    {
+      "name": "Follow me on Instagram",
+      "url": "https://www.instagram.com/shreyaprasad_/",
```
Comment (kabir0x23) [inline public/data/ShreyaPrasad1209.json:29]: The code looks good ✅.
I have a one doubt: 👇
You should give a look at your Instagram username.
can you confirm which one is correct, the previous one or the below one.
 ```suggestion
       "url": "https://www.instagram.com/shreyacasmalert_/",
```

**Review by kabir0x23 — CHANGES_REQUESTED** [review]

File: public/data/ShreyaPrasad1209.json (hunk 1 above)
Comment (ShreyaPrasad1209) [reply public/data/ShreyaPrasad1209.json]: Both urls are incorrect, I've updated it in my latest commit.

**Review by kabir0x23 — APPROVED** [review]

File: public/data/ShreyaPrasad1209.json (hunk 1 above)
Comment (kabir0x23) [reply public/data/ShreyaPrasad1209.json]: > Both urls are incorrect, I've updated it in my latest commit.

reviewed✅.
By the way, the username I have mentioned above is the same one that you have committed.

**Review by Pradumnasaraf — APPROVED** [review]
Looks good to merge. :tada:

__
[sema-logo](https://semasoftware.com/gh) &nbsp;**Summary:** :ok_hand: This code looks good&nbsp; | &nbsp;**Tags:** Reusable

**Pull request merged**